src/js/pages/jobs/JobsTable.js

```diff
@@ -96,7 +100,22 @@ class JobsTable extends React.Component {
     });
   }
 
-  sortJobNames(prop, direction) {
+  getCompareFunctionByProp(prop) {
+    switch (prop) {
+      case "name":
+        return (a, b) => a.name.localeCompare(b.name);
+      case "status":
+        return (a, b) =>
+          JobStates[a.status].sortOrder - JobStates[b.status].sortOrder;
+      case "lastRun":
+        return (a, b) =>
+          JobStatus[a.lastRun.status].sortOrder -
+          JobStatus[b.lastRun.status].sortOrder;
+    }
+  }
+
+  jobSortFunction(prop, direction) {
```

Comment:
Yes I figured I believe that is a bad name convention being used there and would be nice if we stop doing that.

Not that we do that but for example purposes e.g `var tasksArray = []` so why `jobSortFunction ` it seems redundant to me, let's see what others think about it.